build-tools/packages/build-tools/src/common/gitRepo.ts

```diff
@@ -5,9 +5,12 @@
 
 import { parseISO } from "date-fns";
 import { exec, execNoError } from "./utils";
-
 import registerDebug from "debug";
+import { statSync } from "fs-extra";
+import path from "node:path";
```

Comment:
I thought biome wouldn't like this ordering. I thought all node: imports were to be sorted very early.?

Reply:
ordering is not on for build-tools, see https://github.com/microsoft/FluidFramework/blob/2822c6b8c41c452a8501b92ce78ed9f49e1f4400/build-tools/biome.jsonc#L5

Reply:
Ordering is enabled in https://github.com/microsoft/FluidFramework/pull/20230